_sass/vendor/_bootstrap.scss

```diff
@@ -4609,12 +4609,12 @@ tbody.collapse.in {
 
```

Comment:
Let's avoid modifying vendor files.  If needed, I think we can update bootstrap if this was fixed in a later release however.

Reply:
Okay, so do you want me to close this Pr, should I just try solving some other way

Reply:
I would revert these changes, and add a media query to the needed SCSS file in the assets folder.